README.md

```diff
@@ -426,6 +426,50 @@ Finally pass this plugin into PostGraphile via `--append-plugins` or
 See [src/AggregateSpecsPlugin.ts](src/AggregateSpecsPlugin.ts) for examples and
 more information.
 
+## Disable aggregates
+
+The aggregates are created for all tables which really massively increases the type def in size.
+You can disable aggregates by default and enable them only for the tables you need.
+
+```ts
+// Disable aggregates by default
+graphileBuildOptions: {
+  disableAggregatesByDefault: true,
+}
```

Comment:
People don't necessarily know where to put `graphileBuildOptions`, so here's a slightly more complete example:

```suggestion
const middleware = postgraphile(DATABASE_URL, SCHEMAS, {
  // ...
  appendPlugins: [
    // ...
    PgAggregatesPlugin,
  ],
  
  graphileBuildOptions: {
    // Disable aggregates by default; opt each table in via the `@aggregates` smart tag
    disableAggregatesByDefault: true,
  },
});
```